src/functions/slack-user-avatar/handler.ts

```diff
@@ -80,14 +75,7 @@ const getSlackUserAvatar: ValidatedEventAPIGatewayProxyEvent<any> = async () =>
     }
 
     const images: Response[] = [];
-    for (const timebankUser of timebankUsers) {
-      const emailPrefix = timebankUser.email.split("@")[0];
-      const slackUser = slackUsersCache.users.find(user => user.name === emailPrefix);
-      if (slackUser) {
-        images.push({ personId: timebankUser.id, image_original: slackUser.profile.image_original });
-      }
-    }
```

Comment:
Lets comment out this code (rather than delete) and leave a TODO: comment that it needs to be updated to use SeveraUsers. It might be a useful reference for whoever implements this feature in the future.